hadoop-ozone/ozone-manager/src/main/resources/webapps/ozoneManager/ozoneManager.js

```diff
@@ -113,5 +113,12 @@
         require: {
             overview: "^overview"
         },
+        controller: function ($http) {
+            var ctrl = this;
+            $http.get("http://localhost:9874/jmx?qry=Ratis:service=RaftServer,group=*,id=*")
```

Comment:
Do we need to specify the localhost here ? I mean the fully qualified path?

Reply:
I forgot to remove it !!
Thanks for the find